ansible/install_exp_env_centos.sh

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env bash
```

Comment:
Why not just a symbolic link to https://github.com/intelsdi-x/swan/blob/master/vagrant/provision_experiment_environment.sh ?

Reply:
Because of changed swan services paths.

Reply:
@Dual2Core I don't think it's a good idea. You copied *whole* script while you could *parametrise* existing one and use it.

Reply:
So parametrize it :P

Reply:
Or just symlink Swan's `vagrant` directory to `/vagrant`.